cmake/toolchain/espressif/target.cmake

```diff
@@ -8,9 +8,10 @@ set(COMPILER gcc)
 set(LINKER ld)
 set(BINTOOLS gnu)
 
-set(CROSS_COMPILE_TARGET_xtensa_esp32    xtensa-esp32-elf)
-set(CROSS_COMPILE_TARGET_xtensa_esp32s2  xtensa-esp32s2-elf)
-set(CROSS_COMPILE_TARGET_riscv_esp32c3   riscv32-esp-elf)
+set(CROSS_COMPILE_TARGET_xtensa_esp32     xtensa-esp32-elf)
```

Comment:
This isn't required anymore.  
I'll submit PR removing espressif's toolchain cmake content soon.